setup/unattended-install/unattended-install.php

```diff
@@ -310,8 +320,16 @@ function PrintUsageAndExit()
 	}
 	else
 	{
-		$oMysqli = new mysqli($sDBServer, $sDBUser, $sDBPwd);
-		if (!$oMysqli->connect_errno)
+		$oMysqli = new mysqli();
+		$iMySqlFlag = 0;
+		if ($bDBTlsEnabled)
+		{
+			$iMySqlFlag = (empty($sDBTlsCa))
+				? MYSQLI_CLIENT_SSL_DONT_VERIFY_SERVER_CERT
+				: MYSQLI_CLIENT_SSL;
+			$oMysqli->ssl_set($bDBTlsEnabled, null, $sDBTlsCa, null, null);
+		}
+		if (!$oMysqli->real_connect($sDBServer, $sDBUser, $sDBPwd, null, null, null, $iMySqlFlag))
```

Comment:
```suggestion
		if ($oMysqli->real_connect($sDBServer, $sDBUser, $sDBPwd, null, null, null, $iMySqlFlag))
```
Oops mistake from my end..